articles/macos-mdm-setup.md

```diff
@@ -53,7 +52,52 @@ If no default team is set for a host platform (macOS, iOS, or iPadOS), then newl
 
 > A host can be transferred to a new (not default) team before it enrolls. In the Fleet UI, you can do this under **Settings** > **Teams**.
 
-### Simple Certificate Enrollment Protocol (SCEP)
+## Volume Purchasing Program (VPP)
+
+> Available in Fleet Premium
+
+To connect Fleet to Apple's VPP, head to the guide [here](https://fleetdm.com/guides/install-vpp-apps-on-macos-using-fleet).
+
+### Best practice
+
+Most organizations need only one ABM MDM server and one VPP token to manage their devcices and content.
```

Comment:
Most organizations need only one ABM MDM token and one VPP token to manage their devices and content.

Reply:
What do we mean by "content" here?

Reply:
Removed it for now to keep the tone focused on the goal: managing devices (aka hosts)

Reply:
@noahtalerman "Content" as in "Content Manager" in Apple Buisness Manager. This is how Apple refers to Apps & Books.